get_dir.c

```diff
@@ -0,0 +1,38 @@
+#include "shell.h"
+#include <dirent.h>
+
+char **get_dir(char *path)
+{
+    struct dirent *entry;
+    DIR *dir_pointer;
```

Comment:
A pointer to `DIR` object represents a _directory stream_.
A directory stream is a sequence of directory entries. 
Directory entries (`struct dirent`) contain: name of file/directory, type (regular, directory, symbolic link), etc.

| method | behaviour | gotchas|
| ---- | ---- | --- | 
| `opendir` | returns a pointer to a directory stream | ?|
| `readdir` |  returns the next object in stream (a pointer to `struct dirent`)| returns `NULL` if error or no more entries|
| `closedir` | closes the directory stream | ? |